### DISCUSSION TITLE
ABC 5 Minneapolis MN (KSTP)

### DISCUSSION THREAD

**Discussion by jhedfors** [iptv-org/community#737] [Q&A]
**Channel Name:** ABC 5 Minneapolis MN (KSTP) 
**Channel ID:** KSTPDT1.us
**Country:** USA
**Language:** English
**Website:** https://kstp.com/livevideo/
**Stream URL (optional):**

**Comment by Sphinxroot** [comment]
https://dcs-live.mp.lura.live/server/play/zA65M4DuXVnxibP0/manifest.m3u8? + token need to be streamlinked

**Comment by BellezaEmporium** [comment]
> https://dcs-live.mp.lura.live/server/play/zA65M4DuXVnxibP0/manifest.m3u8? + token need to be streamlinked

Anvato-like stream. Token can be gathered by looking after the scripts.

**Comment by jhedfors** [comment]
I am curious. What needs to happen next in the process to get this channel working?

**Comment by BellezaEmporium** [comment]
https://tkx.mp.lura.live/rest/v2/mcp/video/adstwR1j9Z7P8Ynm?anvack=rz93EA03Zenj1h23jMIYZC6DVMVQ8DMe

Video link is in there.

**Comment by jhedfors** [comment]
> https://tkx.mp.lura.live/rest/v2/mcp/video/adstwR1j9Z7P8Ynm?anvack=rz93EA03Zenj1h23jMIYZC6DVMVQ8DMe
> 
> Video link is in there.

Thank you. I assume there must be some sort of translation step that needs to happen to get this video link to work in the playlist?

**Comment by BellezaEmporium** [comment]
> > https://tkx.mp.lura.live/rest/v2/mcp/video/adstwR1j9Z7P8Ynm?anvack=rz93EA03Zenj1h23jMIYZC6DVMVQ8DMe
> > Video link is in there.
> 
> Thank you. I assume there must be some sort of translation step that needs to happen to get this video link to work in the playlist?

Indeed, we simply read the manifest URL from the script and once we have it, we make an automatic redirection to the stream URL.

**Comment by BellezaEmporium** [comment]
Please try http://lanesh4d0w.codes/services_page/fx_mnp.html

**Comment by jhedfors** [comment]
> 



> Please try http://lanesh4d0w.codes/services_page/fx_mnp.html

The player does not seem to understand how to handle the .html extension, as I think it is expecting .m3u8.  It does not get to the point of redirecting.  I have tried both VLC and the IPTV player "OTT Navigator".

**Comment by BellezaEmporium** [comment]
> The player does not seem to understand how to handle the .html extension, as I think it is expecting .m3u8. It does not get to the point of redirecting. I have tried both VLC and the IPTV player "OTT Navigator".
> 

I'm trying to see how I can handle it.

**Comment by jhedfors** [comment]
> > The player does not seem to understand how to handle the .html extension, as I think it is expecting .m3u8. It does not get to the point of redirecting. I have tried both VLC and the IPTV player "OTT Navigator".
> 
> I'm trying to see how I can handle it.

Thanks for your help! I can confirm that when opened **with Chrome,** the link that it then downloads works.

**Comment by jhedfors** [comment]
I am guessing that you are trying to implement a better way than what was done previously.
https://github.com/iptv-org/iptv/issues/4348

**Comment by BellezaEmporium** [comment]
> I am guessing that you are trying to implement a better way than what was done previously. https://github.com/iptv-org/iptv/issues/4348

I actually took it back from my service. The only thing I've optimized is the link.

**Comment by jhedfors** [comment]
Really missing this channel. Any updates on this?

**Comment by freearhey** [comment]
Closed in favor of https://github.com/iptv-org/iptv/issues/19739
